stdlib/map.gr

```diff
@@ -8,115 +8,106 @@ import WasmI32 from "runtime/unsafe/wasmi32"
 import { allocateArray } from "runtime/dataStructures"
 
 // TODO: Consider implementing this as List<(Box<k>, Box<v>)>
-record Bucket<k, v> {
-  mut key: k,
-  mut value: v,
-  mut next: Option<Bucket<k, v>>
-}
```

Comment:
Always define records over multiple lines for easier comprehension?

Reply:
👍